src/qz/printer/action/PrintPDF.java

```diff
@@ -223,13 +223,18 @@ public void print(PrintOutput output, PrintOptions options) throws PrinterExcept
                     rotatePage(doc, pd, pxlOpts.getRotation());
                 }
 
+                boolean landBased = pxlOpts.getOrientation() != null && pxlOpts.getOrientation() != PrintOptions.Orientation.PORTRAIT;
```

Comment:
Can we explain in the code why tracking landBased is important and why we have an exception to this calculation below?